tekton/Makefile

```diff
@@ -13,7 +14,7 @@ build: app.yaml
 app.yaml:
```

Comment:
Running `make app.yaml` shows quite a lot of changes

Reply:
I usually prefer to tie a release to a specific version so I can run my mini test suite against it - as well as ensure the `manifest.yaml` that describes the app has the correct version number in it (with one of the checklist items being "Have you included the specific version of the application to be installed?" in the PR template, I assumed you wanted marketplace apps to be tied to specific versions if possible)

I could re-engineer this to automatically update to the "latest" version, although I'd want to also include tests against the test suite - which could be done.  My original though of going with the LTS versions of Tekton was to avoid the constant upgrade cycle, although with enough testing it could be done.

Reply:
Also jumped from 0.24 to 0.44, so probably explains a *lot* of the diff updates.  I had to include metrics-service as a pre-requisite as they are now tekton installs a HPA by default for scaling.

Reply:
Thanks for the information, I created Updatecli to "quickly" implement automated update scenario in none standard environment using a declarative approach.
So I am usually quite interested to understand how third project handle that project. Where the information is coming from and what should be automatically updated,etc.

In this case, I was curious where that LTS information came from but based on your input I don't see a robust way to get that information automatically from that markdown without a human reading it.